### PR TITLE
🏗 Optionally exclude JSS with `make-extension --nojss`

### PR DESCRIPTION
**Pull request by alanorozco**
Since they require CSS on the AMP layer, but not JSS on the component, I'd like to specify on a guide that Bento video players can be started with:

```console
amp make-extension --bento --nojss
```

### CODE REVIEW

**Review by dmanek — APPROVED** [review]

**Review by rcebulko — APPROVED** [review]

**Review by caroqliu — APPROVED** [review]
WDYT about adding `BaseElement['shadowCss'] = COMPONENT_CSS;` in the generated `base-element.js`?

**Comment by alanorozco** [comment]
@caroqliu That's already in place. 

https://github.com/ampproject/amphtml/blob/3592c8c0676f5e73e6b8ab1248a57119cff85d93/build-system/tasks/make-extension/template/bento/extensions/amp-__component_name_hyphenated__/__component_version__/base-element.js#L39-L40

We do not cleanup the following using `--nojss`, so we're relying on good review eyes to make sure we don't submit the following. We could alternatively move this section into the JS conditional, but it might be too much code outside the template.

```diff
- /** @override */ 
- BaseElement['shadowCss'] = null;
```

**Comment by rcebulko** [comment]
> We do not cleanup the following using `--nojss`, so we're relying on good review eyes to make sure we don't submit the following. We could alternatively move this section into the JS conditional, but it might be too much code outside the template.
@alanorozco What do we want to ensure is not submitted? We do have the `__do_not_submit__` token

**Comment by alanorozco** [comment]
@rcebulko Added `do not submit` blurb.

File: build-system/tasks/make-extension/template/bento/extensions/amp-__component_name_hyphenated__/__component_version__/base-element.js [hunk 1]
```diff
@@ -36,5 +36,9 @@ BaseElement['layoutSizeDefined'] = true;
 /** @override */
 BaseElement['usesShadowDom'] = true;
 
+// DO NOT SUBMIT: If BaseElement['shadowCss']  is set to `null`, remove the
```
Comment (rcebulko) [inline build-system/tasks/make-extension/template/bento/extensions/amp-__component_name_hyphenated__/__component_version__/base-element.js]:
```suggestion
// __do_not_submit__: If BaseElement['shadowCss']  is set to `null`, remove the
```
This as-written will probably fail CI

**Review by rcebulko — COMMENTED** [review]
OWNERS bump

**Review by krdwan — COMMENTED** [review]
LGTM

**Pull request merged**